.pre-commit-config.yaml

```diff
@@ -2,4 +2,4 @@ repos:
 - hooks:
   - id: secret-scanner
   repo: git@github.com:snowflakedb/casec_precommit.git
-  rev: v1.5
+  rev: v1.8
```

Comment:
With the old version of the secret-scanner, git commit on my machine is failing with

```
/home/lsembera/.cache/pre-commit/repo47p9zrdc/secret_scanner.sh: line 48: pip: command not found 
/home/lsembera/.cache/pre-commit/repo47p9zrdc/secret_scanner.sh: line 22: detect-secrets: command not found 
/home/lsembera/.cache/pre-commit/repo47p9zrdc/secret_scanner.sh: line 23: [: !=: unary operator expected
```